src/sentry/data_export/processors/base.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+
+from sentry.models import Project
+
+from ..base import ExportError
+
+
+class ExportProcessor:
```

Comment:
also the more I think about it, are there more things that will be shared besides this `get_project()` method? If not, it might make more sense to just have this added back into the `IssuesByTagProcessor` and have that be your base for now `class IssuesByTagProcessor(object):`

Reply:
Yep, makes sense -- removed the BaseProcessor in 8c6bebc